ResearchKit/Consent/ORKConsentLearnMoreViewController.h

```diff
@@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ORKConsentLearnMoreViewController : UIViewController
 
 - (instancetype)initWithHTMLContent:(NSString *)content;
+- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
```

Comment:
Would you like me to change it to NSURL instead of the request?

J

Sent from my iPhone 6plus

> On May 22, 2015, at 3:56 PM, jwe-apple <notifications@github.com> wrote:
> 
> In ResearchKit/Consent/ORKConsentLearnMoreViewController.h:
> 
> > @@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
> >  @interface ORKConsentLearnMoreViewController : UIViewController
> >  
> >  - (instancetype)initWithHTMLContent:(NSString *)content;
> > +- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
> I don't think it's a good idea to take something that's not a file URL here... we have no loading indication or navigation controls in this view.
> 
> —
> Reply to this email directly or view it on GitHub.
>